librocksdb_sys/build.rs

```diff
@@ -99,8 +104,10 @@ fn build_rocksdb() -> Build {
         .register_dep("SNAPPY")
         .define("WITH_SNAPPY", "ON")
         .build_target("rocksdb")
+        .very_verbose(true)
```

Comment:
Is it necessary?

Reply:
I find it helpful for debugging, and it doesn't change `cargo build` output without `-vv`.